src/components/common/TmSessionWelcome.vue

```diff
@@ -12,6 +12,13 @@
 
       <div class="tm-session-main">
         <LiSession
+          v-if="!browserWithLedgerSupport"
+          icon="usb"
+          title="Sign in with Ledger Nano S"
```

Comment:
This is not important, but perhaps a non-breaking space between `Nano` and `S`?

<img width="368" alt="Screenshot 2019-05-07 at 17 30 46" src="https://user-images.githubusercontent.com/360865/57312477-ef8bfe00-70ed-11e9-9d70-d281bd201e11.png">
vs.

<img width="366" alt="Screenshot 2019-05-07 at 17 30 58" src="https://user-images.githubusercontent.com/360865/57312485-f286ee80-70ed-11e9-85ae-e64dceb814f7.png">

It's just the first thing I saw, and it bothered me, but the nbsp doesn't look that great either. Am I being too nit picky @jbibla ? :)

*Or perhaps change the total width.

Reply:
@colw Yeah, I don't like how this looks. I'll change it.

Reply:
no such thing as _too_ nit picky 😉 

`nbsp;` is a great idea.

Reply:
😆 

Also, I should have been clear this was taken on the iPhone 8 (simulator).

Reply:
If you do go with a `nbsp;` it would be a a good idea to make it obvious it's there, either through a unicode character, or even a small comment.

Reply:
@colw Thanks for the help! Looking better for me now.